### ISSUE TITLE
Add support for Playwright

### ISSUE THREAD

**Issue by Lonli-Lokli** [trion-development/docker-ng-cli-e2e#13]
Add support for Playwright
https://github.com/microsoft/playwright

**Comment by everflux** [comment]
Thanks for raising this issue. 
I am not sure that playwright would be a good fit for this image. The idea was to offer the canonical tools for angular based development as Docker images especially for ci builds. 

How would you use playwright in your build? How do you feel about using a separate image that provides playwright in your build?

**Comment by Lonli-Lokli** [comment]
@everflux Unfortunately Protractor seems to be abandoned https://github.com/angular/protractor/commits/master

So I'm looking for another options for angular apps, like cypress or more promising Playwright.
Actually I will it use similar like it's now with Protractor - Angular CLI based app, set of scripts and new E2E tooling.
I do not have enought knowledge as of now to create such image but IMO it's completely ok to have it in a separate image.

**Comment by everflux** [comment]
I don't think that protractor is abandoned, and can't conclude that from the commits to master which show recent progress.
If you want to use a different tool for e2e tests, than protractor, for example puppeteer, cypress or playwright I suggest that you use a separate image in your CI pipeline.
For example use trion/ng-cli(-karma) for building and unit testing and a cypress image for e2e tests.

I could not find available playwright images on dockerhub, I will look into providing a build. But this would be separate from the Angular tooling images. Would that work for you?

**Comment by Lonli-Lokli** [comment]
There are 17 commits in 2020 (I believe that's just because Angular drops support for older Node), 2 commits in 2019 and 43 commits in 2018 - as for me it's pretty close to not-maintained project :)

Yep, I saw your images, thanks for them! As of now I'm going to use Jest for unit testing and cypress\playwright for e2e.

Separate image would be totally fine!
